src/azure-cli/azure/cli/command_modules/servicefabric/_params.py

```diff
@@ -298,6 +314,9 @@ def load_arguments(self, _):  # pylint: disable=too-many-statements
         c.argument('vm_image_version', help='The version of the Azure Virtual Machines Marketplace image. ', default='latest')
         c.argument('capacity', arg_type=capacity)
         c.argument('placement_property', arg_type=placement_property)
+        c.argument('is_stateless', arg_type=get_three_state_flag(), help='Indicates if the node type can only host Stateless workloads.', default=False)
```

Comment:
```suggestion
        c.argument('is_stateless', arg_type=get_three_state_flag(), help='Indicate if the node type can only host Stateless workloads.', default=False)
```

Reply:
I think the correct conjugation is with s in this case